x-pack/legacy/plugins/apm/server/lib/transactions/charts/get_timeseries_data/index.ts

```diff
@@ -11,8 +11,8 @@ import { timeseriesTransformer } from './transform';
 
 export async function getApmTimeseriesData(options: {
   serviceName: string;
-  transactionType?: string;
-  transactionName?: string;
+  transactionType: string | undefined;
+  transactionName: string | undefined;
```

Comment:
This is just to ensure that we don't forget to pass the required arguments to the function.